### DISCUSSION TITLE
Cannot escalate form SOC to TheHive "Request failed with status code 400"

### DISCUSSION THREAD

**Discussion by DDD-Digithead** [Security-Onion-Solutions/securityonion#2462] [Unsupported Versions] [answered]
I came across Security Onion some time again and have had it up and running for a while to get familiar with it. However after the most recent update using soup, alerts can no longer be escalated to TheHive. Any attempt to do so results in a "Request failed with status code 400" error. Using so-status indicates that all services are indeed running.

Being a "noob" here, can someone please point me in the right direction as to where to start digging for the issue? I went so far as to completely reload from scratch with the latest version (as of 12/26/2020) the issue persists.

Thank you in advance.

**Comment by jertel** [comment, marked as answer]
Hi. The /opt/so/log directory is a good place to start. Review both the soc and thehive log files when you attempt the escalation. One of those should give you a little more information about what's going on.

**Reply by DDD-Digithead** [reply]
Thank you. Looking at the log for TheHive revealed nothing at all however while running a tail -f on the SOC log it threw this error when attempting to escalate an alert to TheHive:

timestamp=2020-12-27T15:43:02.48991592Z level=error message="Failed request" contentLength=0 elapsedMs=49 error="Request did not complete successfully (401): 401 Unauthorized" impl=*server.CaseHandler method=POST path=/api/case 

It appears that there may be a key mismatch somewhere perhaps.(?)

**Reply by jertel** [reply]
Sounds similar to this recent discussion. Take a look and see if it gets you further. https://github.com/Security-Onion-Solutions/securityonion/discussions/2337

**Reply by DDD-Digithead** [reply]
No, that's not it either. Although I had indeed created a second user in TheHive, attempting to escalate never worked from the start (after a complete reload of the appliance). I had tried to create that second user to see if that would fix the problem but it did not.

I made a backup copy of "global.sls", logged into TheHive as the administrator, copied the API key and placed it into "global.sls" and restarted TheHive. But after that I couldn't log into TheHive web interface at all and had to put the key back the way it was.

**Reply by DDD-Digithead** [reply]
Wow! Just for laugh's I decided to try copying the Admin user's API key from TheHive user administration page and put it back into "global.sls", restart TheHive using "so-thehive-restart" and then just giving it some time to 'digest'. Later I restarted the entire appliance and allowed all of the services to start up and then attempted another "Escalate" from the "Alerts" listing and it worked.

Looks like that solved the problem after all.
